Cakes-Co/index.html

```diff
@@ -25,7 +25,7 @@
             <a href="">About</a>
           </nav>
         </aside>
-        <p class="motto-p">The best cakes in town delivered to your door</p>
+        <p class="motto-p">The most delicious cakes in town delivered to your door</p>
```

Comment:
I don't think this change belongs in this PR. Only changes inside the wireframe folder belong in this PR.

Reply:
Sorry, this was my bad